classes/main.rb

```diff
@@ -1,22 +1,37 @@
-require_relative 'student'
-require_relative 'classroom'
-require_relative 'person'
-require_relative 'rental'
-require_relative 'book'
+require_relative 'app'
 
-# Create Instances
+def display_ui
+  puts 'Please choose an action by entering a numbe:'
+  puts '1 - list all book'
+  puts '2 - list all people'
+  puts '3 - create a person'
+  puts '4 - create a book'
+  puts '5 - create a person'
+  puts '6 - list all rentals for a given person id'
+  puts '7 - exit'
+  user_choice = gets.chomp.to_i
```

Comment:
Please kindly change option number 5 to create rental not create person you already have create person on option number 3.